### ISSUE TITLE
Tidying up

### ISSUE THREAD

**Issue by willtebbutt** [MichielStock/Kronecker.jl#25]
There's quite a lot of stuff in the repo that isn't strictly code-related. e.g. the notebooks, tex, and apparently there's some Ruby code somewhere? Is there any chance that these could be moved to a separate repo and linked to, rather than included in this repo? i.e. it would be nice if this were 100% Julia code, and all auxiliary stuff were elsewhere.

(Sorry for raising yet another slightly pedantic issue...)

**Comment by MichielStock** [comment]
You are right, but for Juliacon I had to add the paper and all files to build it into the repo. The notebook is for the moment a poor man's replacement for documentations. 

When the Juliacon publishes the extended abstracts, I will remove the .tex files. I also hope to have a proper doc page soon, so then I will remove the notebooks!

(and the style-related remarks are appreciated)

**Comment by willtebbutt** [comment]
Ahhh okay, thanks for letting me know. Will leave this open until that happens then.

**Comment by MichielStock** [comment]
Documentation is now online. I will remove the notebook.

**Comment by willtebbutt** [comment]
Does this mean that the tex and ruby stuff will disappear also?

**Comment by MichielStock** [comment]
When the Juliacon paper is accepted, promised!

**Comment by willtebbutt** [comment]
Oh, they've still not released the proceedings?
